docs/DeveloperGuide.md

```diff
@@ -1195,12 +1215,24 @@ otherwise)
 
 **Extensions:**
 
-- 1a. Logged-in Owner does not specify the required field(s)
+- 1a. Logged-in Owner does not specify the required field(s).
+
+    - 1a1. CMS displays an error to Logged-in Owner to key in all the fields required to create a customer.
+
+      Use Case ends.
+
+- 1b. Logged-in Owner specifies duplicated fields.
```

Comment:
Should this be standardised with the one above?

Reply:
Do you mean Logged-in Owner types duplicated fields from UC02?